ligo/haskell/src/Ligo/Typescript.hs

```diff
@@ -0,0 +1,380 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+module Ligo.Typescript (generateTs) where
+
+import Universum
+
+import Data.Char (toUpper)
+import qualified Data.Text as T
+import Fmt (Buildable(build), blockListF', (+|), (|+))
+import qualified Fmt
+import System.FilePath
+
+import Lorentz.Entrypoints
+import Michelson.Typed.Annotation
+import Michelson.Typed.Entrypoints (pnNotes)
+import Michelson.Untyped.Annotation
+
+-- | This is mostly same as the 'Notes t' type, but without the type
+-- parameter and without annotatations that we are not interested.
```

Comment:
Replace `annotatations` with `annotations`.

Reply:
I have fixed this and couple of other typos in comments.